### PR TITLE
remove torchsearchsorted dependency

### PR DESCRIPTION
**Pull request by Nikronic**
Hi,
To remove aforementioned dependency, PyTorch built-in method has been used. Although, it has a small bug which can be resolved by detaching tensors with grad as the output of `torch.searchsorted` is `int` so it does not have gradients.

Also, the readme has been updated regarding this change, and a typo has been fixed.

Furthermore, I added my experience with model for training on `fern` dataset (time wise).

### CODE REVIEW

File: run_nerf_helpers.py [hunk 1]
```diff
@@ -223,7 +221,7 @@ def sample_pdf(bins, weights, N_samples, det=False, pytest=False):
 
     # Invert CDF
     u = u.contiguous()
-    inds = searchsorted(cdf, u, side='right')
+    inds = torch.searchsorted(cdf.detach(), u, right=True)
```
Comment (Nikronic) [inline run_nerf_helpers.py:224]: Even though method should be ok with tensors with enabled grads, but it seems it is a known bug and one has to detach tensors or set `requires_grad=False` to use this method.
Please see this issue: 
https://github.com/pytorch/pytorch/issues/41389

**Comment by yenchenlin** [comment]
Awesome contribution, thanks 🎉 ! Merged.
